adminer/content.md

```diff
@@ -0,0 +1,27 @@
+# Adminer
+
+## What is Adminer?
+
+Adminer (formerly phpMinAdmin) is a full-featured database management tool written in PHP. Conversely to phpMyAdmin, it consist of a single file ready to deploy to the target server. Adminer is available for MySQL, PostgreSQL, SQLite, MS SQL, Oracle, Firebird, SimpleDB, Elasticsearch and MongoDB.
```

Comment:
It would need additional extensions. At the very least MS SQL needs an extension. Thanks for the heads up, I'll check.